packages/app/src/App.tsx

```diff
@@ -317,7 +318,7 @@ function App(props: { flameFromQuery?: FlameFunction[] }) {
                 produce((flames) => {
                   flames.push({
                     probability: 0.1,
-                    color: vec2f(),
+                    color: { x: 0, y: 0 },
```

Comment:
This was a bug actually, we need to have object as color, vec2f returns array in JS, e.g., [0, 0], so the png round trip would fail if user were to add a flame from UI and export it as such

Reply:
Interesting, yeah.